core/strings/strings.odin

```diff
@@ -1713,10 +2870,15 @@ right_justify :: proc(str: string, length: int, pad: string, allocator := contex
 
 	return to_string(b)
 }
+/*
+Writes a given pad string a specified number of times to an io.Writer
 
-
-
-
+**Inputs**  
+- w: The io.Writer to write the pad string to
+- pad: The pad string to be written
+- pad_len: The length of the pad string
```

Comment:
Private function used by `*_justify`. `pad_len` and `remains` are rune counts, not byte counts.